### PR TITLE
John add heading component

### PR DESCRIPTION
**Pull request by JohnZ9865**
trust me it works.
image

### CODE REVIEW

File: src/components/Heading.jsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
+import Image from "next/image";
+import photo from "public/colored-graph.svg";
 const Heading = () => {
-  return <div>Heading</div>;
+  return <Image src={photo} />;
```
Comment (pondohoo) [inline src/components/Heading.jsx:4]: can you pass in the heading text as props and have it display next to the image. also include the little yellow bar below the text.

check out the image in the issue for reference

File: src/components/Heading.jsx [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+        <div className="" style={{ paddingRight: "10px" }}>
+          <Image src={photo} />
+        </div>
+        <h2 className="">{text}</h2>
+      </div>
+
+      <div className="flex">
+        <div className="px-10"></div>
+        <div className="px-4"></div>
+        <div className="bg-orange-500 px-4 py-1"> </div>
```
Comment (pondohoo) [inline src/components/Heading.jsx:16]: remember use the colors in tailwind.config.js not the built in colors

File: src/app/about/page.js [hunk 3]
```diff
@@ -1,9 +1,11 @@
 import Banner from "@/components/Banner";
+import Heading from "src/components/Heading.jsx";
 
 const about = () => {
   return (
     <div>
       <Banner />
+      <Heading text="Join HLG@UCR" />
```
Comment (pondohoo) [inline src/app/about/page.js:8]: we can leave this honestly but make the text "Who We Are" cuz it's the about page

File: src/components/Heading.jsx (hunk 2 above)
Comment (pondohoo) [inline src/components/Heading.jsx:16]: make the edges rounded for the orange block like in the image

**Comment by JohnZ9865** [comment]
fixed

**Review by pondohoo — APPROVED** [review]

**Pull request merged**